bowtie/report/public/index.html

```diff
@@ -0,0 +1,29 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Bowtie –</title>
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-KK94CHFLLe+nY2dmCWGMq91rCGa5gtU4mk92HdvYe+M/SXH301p5ILy+dN9+nJOZ"
+      crossorigin="anonymous"
+    />
+  </head>
+
+  <body>
+    <div id="root"></div>
+
+    <script
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/js/bootstrap.bundle.min.js"
+      integrity="sha384-ENjdO4Dr2bkBIFxQpeoTz1HIcje39Wm4jDKdf19U8gI4ddQ3GYNS7NTKfAdVQSZe"
+      crossorigin="anonymous"
+    ></script>
+
+    <script>
+      const script = document.createElement("script");
+      script.src = "https://cdn.jsdelivr.net/npm/pako@2.0.3/dist/pako.min.js";
+      document.head.appendChild(script);
```

Comment:
I guess this lib could be moved to `package.json` too, but I feel like we shouldn't really be unpacking gzips on frontend? I think probably that build/deploy process should take care of unpacking

Reply:
At this stage, the UI is fetching data from the local bowtie webpage (`https://bowtie-json-schema.github.io/bowtie/`${dialect_version}`.json.gz`). Also we are 6 **_dialects_** (`2020-12`, `2019-09`, `7`, `6`, `4`, `3`) --- that means 6 data files. I think it would be more feasible to dynamically fetch the data based on user's choice of **_dialect_** (to keep the app lightweight) rather than collecting all the data at once and bundling them in our app. However, let me know what do you think.

Reply:
I'm only suggesting that if we need this data to be unpacked we could have it uploaded in unpacked version. Currently, each user's browser is doing the unpacking process (affecting user experience I guess? But i don't know how much time does this process take), when it could be done just once during building/deployment or at any stage it is convenient

Reply:
The original reason to investigate was that the unpacked data for each draft is ~10MB or so (#146), and the compressed data is ~50KB. so it was a noticeable reduction in load time I think. But we can reconsider of course (it'd definitely be nice if GitHub supported Content-Type and then we'd obviously get the browser automatically decompressing the gzip before the app saw it)

Reply:
Hmm, that is what I was thinking - the GH pages should expose static files which are **NOT** gzipped, but the underlying server should handle their compressing if client requires so. And the browser should unpack it automatically. So we could use URL like `https://bowtie-json-schema.github.io/bowtie/draft2020-12.json` (without gzip extension), and if client sends request with header `Accept-Encoding=gzip` it should receive gzipped response (with response header `Content-Encoding=gzip`). It looks like e.g. getting bowtie HTML via https://bowtie-json-schema.github.io/bowtie is working exactly like this:

- without `Accept-Encoding` header: ~8.6mb
- with `Accept-Encoding` header: ~200kb

Haven't had time to read the whole discussion from #146 so maybe I missed some important details. Let me know if this approach makes sense

Reply:
Ya, that's totally reasonable. It indeed increased the loading speed!

Reply:
Ya, that's totally reasonable. Till now, I too was thinking that pages didn't support `Accept-Encoding`